projects-evaluation-criteria.md

```diff
@@ -0,0 +1,60 @@
+
+### Mandatory checks
+
+- [ ] Makefile includes the following targets:
+  - [ ] all (default build)
+  - [ ] clean
+  - [ ] test
+- [ ] Project builds without warnings with -Wall -Wextra flags
+- [ ] Project compiles on the target system
+	- [ ] provided remote computer
+	- [ ] Docker container provided by project author
+- [ ] All dependencies are documented in README
+- [ ] Automated tests are present
+- [ ] Tests cover core functionality
+- [ ] All tests pass successfully
+- [ ] README.md exists and contains:
+  - [ ] Project description
+  - [ ] Build instructions
+  - [ ] Usage examples
+  - [ ] List of dependencies
+  - [ ] List of main features (with links to issue tracker)
+- [ ] Project features are tracked in Issues:
+  - [ ] Minimum 3 main features described
+  - [ ] Each feature has acceptance criteria
+- [ ] Each main feature is implemented:
+  - [ ] Feature 1 meets acceptance criteria
+  - [ ] Feature 2 meets acceptance criteria
+  - [ ] Feature 3 meets acceptance criteria
+- [ ] Error handling is implemented for:
+  - [ ] File operations
+  - [ ] Memory allocation
+  - [ ] System calls
+  - [ ] User input
+- [ ] Program exits gracefully in all cases
+- [ ] The final implementation is submitted before deadline as a pull request into github classroom repo. Submission includes:
+  - [ ] Source code
+  - [ ] Makefile
+  - [ ] Tests
+  - [ ] Documentation (at least README.md)
+  - [ ] No binary or other generated files in repository
+
+
+### Optional, but recommended checks
```

Comment:
I'd suggest to add minimal code style, like
1. Using curvy brackets in `for`, `while`, `if`, etc
2. Using tabs or spaces in code blocks